src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseViewInfo.cs

```diff
@@ -0,0 +1,18 @@
+
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#nullable disable
```

Comment:
This should be easy enough to enable nullable stuff for - probably just initialize them to an empty array right?